docs/installation.rst

```diff
@@ -0,0 +1,118 @@
+.. meta::
+   :description: HSA runtime implementation
+   :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _installation:
+
+====================
+Installation
+====================
+
+This document provides information required to build and install ROCR using prebuilt binaries or from source.
+
+Build and install using prebuilt binaries
+-------------------------------------------
+
+Here is how you can install ROCR using prebuilt binaries.
+
+Prerequisites
+*******************
+
+- A system supporting ROCm. See the `supported operating systems <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html#supported-operating-systems>`_.
+
+- Install ROCm. See `how to install ROCm <>`_.
+
+- Install ``libdrm`` package.
+
+
+The ROCR prebuilt binaries include:
+
+**Core runtime package:**
+
+- HSA include files to support application development on the HSA runtime for the ROCR runtime
+
+- A 64-bit version of AMD’s HSA core runtime for the ROCR runtime
+
+**Runtime extension package:**
+
+- A 64-bit version of AMD’s runtime tools library
+
+- A 64-bit version of AMD’s runtime image library
+
+The contents of these packages are installed in ``/opt/rocm/hsa`` and ``/opt/rocm`` by default. The core runtime package depends on the ``hsakmt-roct-dev`` package.
+
+Build and install from source
+--------------------------------
+
+Here is how you can build ROCR from source.
+
+Prerequisites
```

Comment:
@dayatsin-amd Please add commands where applicable under Prerequisites.

Reply:
@dayatsin-amd I have already added the above commands. Should we add commands to install support packages ``libelf-dev`` and ``g++`` and ``xxd``?